src/stories/Card.stories.mdx

```diff
@@ -0,0 +1,121 @@
+import { Meta, Story, Canvas, ArgsTable } from '@storybook/addon-docs/blocks'
+
+import { ThemeProvider } from '../theme'
+import { Card, Button, Flex } from '../components'
+
+import leafImage from '../assets/leafImage.png'
+import testAvatar from '../assets/testAvatar.png'
+
+<Meta title='Data Display/Card' component={Card} />
+
+# Card
+
+O componente Card agrupa informações e ações sobre um mesmo conteúdo em containers flexíveis.
+É composto por subcomponentes e propriedades que podem ou não ser acrescentados à sua estrutura principal, possibilitanto a criação das diversas variações possíveis, de acordo com a preferência do usuário.
+Sendo eles:
+
+- `title`: título.
+- `content`: conteúdo.
+- `subtitle`: subtítulo ou legenda.
+- `avatar`: avatar ou ícone.
+- `media`: mídia.
+- `avatarPosition`: define a posiçao do avatar em relação à imagem da área de mídia.
+- `headerPosition`: define a posição do header, que é a parte que contém o avatar, o título e o subtítulo, entre `row` e `column`.
+
+# General
+
+Componente Card que recebe por `props` todos os elementos mencionados acima.
+
+<Canvas>
+  <Story
+    name='General'
+    parameters={{
+      design: {
+        type: 'figma',
+        url: 'https://www.figma.com/file/O3bKxIcsj2rc1FNIRclJyT/Saturn-System?node-id=806%3A11347'
+      }
+    }}
+  >
+    <ThemeProvider>
+      <Card
+        title='Title here'
+        subtitle='Complementary text'
+        media={leafImage}
+        avatar={testAvatar}
+        content='Text goes here. Lorem ipsum dolor sit amet, consectur adipiscing elit.'
+      />
+    </ThemeProvider>
+  </Story>
+</Canvas>
+
+# General 2
+
+Componente Card completo, com adição da prop `avatarPosition`, que define a posição do avatar e de botões para ações, que podem ser adicionados a gosto do usuário.
+No caso desse exemplo, a imagem do avatar se sobrepõe à imagem contida no media.
+
+Ainda nesse exemplo, é demonstrada a possibilidade de adicionar botões para ações, de acordo com a preferência do usuário.
```

Comment:
Sobre essa parte dos botões acho que tu pode falar um pouquinho mais sobre o componente receber children e poder renderizar o que for passado para ele de forma dinâmica também